test/datatable/demo/test.js

```diff
@@ -848,11 +848,54 @@ define([
                         },
                         {
                             id: 'remove',
-                            label: 'Remove',
+                            label: function() {
+                                if (this.login && this.login.length) {
+                                    return 'Delete account';
+                                }
+                                return 'Remove user';
+                            },
                             icon: 'bin',
+                            action: function(id, row) {
+                                var dialogConfig = {
+                                    message: 'Deletion is not supported yet!',
+                                    content: 'That would affect user id #' + id,
+                                    buttons: 'ok',
+                                    autoRender: true,
+                                    autoDestroy: true
+                                };
+                                if (row.login && row.login.length) {
+                                    dialogConfig.message = 'Account deletion is not supported yet!';
```

Comment:
Messages in both cases are similar. In this case, what purpose is this condition serving?

Reply:
This is a visual playground, in context of testing the feature